eng/pipelines/runtime-extra-platforms.yml

```diff
@@ -42,6 +42,54 @@ jobs:
       isWasmOnlyBuild: ${{ variables.isWasmOnlyBuild }}
       isRollingBuild: ${{ variables.isRollingBuild }}
 
+# Add iOS/tvOS jobs only for rolling builds
+- ${{ if eq(variables.isiOSLikeOnlyBuild, true) }}:
```

Comment:
This condition would mean that this job would never run here, because `variables.isiOSLikeOnlyBuild` would never be true for `runtime-extra-platforms`, IIUC. Should this be checking for rolling build instead?